app/src/main/java/de/tum/in/tumcampusapp/auxiliary/Utils.java

```diff
@@ -121,6 +122,16 @@ public static String getDateTimeString(Date d) {
     }
 
     /**
+     * Checks whether two Dates contain the same day
+     *
+     * @return true if both dates are on the same day
+     */
+    public static boolean isSameDay(Calendar first, Calendar second) {
```

Comment:
Cool function - I think it would be better in https://github.com/TCA-Team/TumCampusApp/blob/c5e37bd616a262741f16afa6ffbaf1448dc1413c/app/src/main/java/de/tum/in/tumcampusapp/auxiliary/DateUtils.java